.github/workflows/build-distribute.yml

```
@@ -0,0 +1,149 @@
+          if [ "${{ inputs.build_type }}" == "debug" ]; then
+            bundle exec fastlane build_and_distribute_debug variant:${{ inputs.variant }}
+          else
+            bundle exec fastlane build_and_distribute_release variant:${{ inputs.variant }}
+          fi
+
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: FLW-Mobile-App - APK(s) ${{ inputs.variant }} generated
+          path: |
```

actionlint (1.7.4)

got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)